README.md

```diff
@@ -421,6 +422,25 @@ Notice: The following `data-uri` attributes
 ```
 would trigger two calls for data because of the difference in query strings, even though the path to service is the same in both.
 
+## Service response status code
```

Comment:
No more full documentation in Readme.md - please move this to documentation/wiki..